processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -149,6 +150,7 @@ public class QueryContexts
   public static final boolean DEFAULT_ENABLE_JOIN_FILTER_PUSH_DOWN = true;
   public static final boolean DEFAULT_ENABLE_JOIN_FILTER_REWRITE = true;
   public static final boolean DEFAULT_ENABLE_JOIN_FILTER_REWRITE_VALUE_COLUMN_FILTERS = false;
+  public static final boolean DEFAULT_QUERY_UNMANAGED_SERVERS = false;
```

Comment:
Rather than a boolean, I think we should use an enum `QueryCloneMode` with values:
```
ONLY: Query ONLY the clones
INCLUDE: Query the clones and normal servers
EXCLUDE (default): Do not query the clones
```

Reply:
Changed.